Jenkinsfile

```diff
@@ -155,6 +164,7 @@ try {
           archiveServerLogs(mongoDbRealmContainer.id, mongoDbRealmCommandServerContainer.id)
           mongoDbRealmContainer.stop()
           mongoDbRealmCommandServerContainer.stop()
+          emulatorContainer.stop()
```

Comment:
I guess this is not set if we are not running the emulator!?

Reply:
Good point 👍